packages/identity/src/identity/der.ts

```diff
@@ -55,6 +55,16 @@ const decodeLenBytes = (buf: Uint8Array, offset: number): number => {
   throw new Error('Length too long (> 4 bytes)');
 };
 
+const decodeLen = (buf: Uint8Array, offset: number): number => {
```

Comment:
We try to keep these libraries as small as possible, because they end up included in frontend bundles and need to be downloaded on every page load in client applications. If the prefix can be represented accurately with a few hard to understand bytes, we will prefer that over including a full library